### DISCUSSION TITLE
Deployment not triggered after changing plan

### DISCUSSION THREAD

**Discussion by jsellam** [vercel/community#1228] [Help] [answered]
### Summary

Hi,

Deployment is well working in my project when I push in the beta branch and since I changed of plan for a "pro trial" the deployment is not triggered when I push on beta branch.
I migrated my project on lobby and the deployment works again.
When I switch back to pro trial again, the deployment does not work

### Additional information

I don't have any informations to know why the deployment is not started.
The source code is well updated and in a commit but don't trigger something in trial pro

The project is based on nextjs template on private github repo (free account)

thx for your help

**Comment by swarnava** [comment, marked as answer]
Can you try disconnecting and reconnect your project from Project Settings > Git?

**Reply by jsellam** [reply]
Hi,
I did it but the problem continue...

**Reply by swarnava** [reply]
Do you have a sample commit SHA?

**Reply by jsellam** [reply]
This commit 00fb17d3611cc2317a814c1337c24b4f92a04636

**Reply by jsellam** [reply]
Hi @swarnava Have you more informations please ? My trial expire in seven days and I can't try it :-(

thx

**Reply by jsellam** [reply]
I also created a totally new project with the nextjs template. The first deployment work well, but preview build on new branch don't start

**Reply by jsellam** [reply]
Ok my email was not set on git on my new computer, sorry

**Reply by swarnava** [reply]
Sorry about the delayed reply and I am glad to hear that it's sorted now. :)
